ooniprobe/View/TestResults/Details/TestDetailsViewController.m

```diff
@@ -49,6 +50,13 @@ - (void)advancedScreens{
                                    actionWithTitle:NSLocalizedString(@"TestResults.Details.RawData", nil)
```

Comment:
I think you can delete the `TODO` comment at the beginning of the function

Reply:
It has been handled in another branch